packages/create-plugin/src/constants.ts

```diff
@@ -99,6 +100,10 @@ export const TEXT = {
   updateNpmDependenciesSuccess: 'Successfully updated the NPM dependencies.',
   updateNpmDependenciesAborted: 'No NPM dependencies have been updated.',
 
+  updateProvisioning: '**Would you like to add provisioning files?**',
+  updateProvisioningSuccess: 'Successfully updated provisioning.',
```

Comment:
Based on the code this locale is used, these are not "updating" as much as "creating".